locale/fi/LC_MESSAGES/django.po

```diff
@@ -332,21 +332,25 @@ msgstr[1] ""
 "\n"
 "%(points)s pistettä"
 
-#: access/templates/access/graded_form_feedback.html:15
+#: access/templates/access/graded_form.html:134
+msgid "neutral option which does not affect grading"
+msgstr "vaihtoehto ei vaikuta kokonaispisteisiin"
```

Comment:
I wonder if the Finnish string should include "neutral": "neutraali vaihtoehto ei vaikuta kokonaispisteisiin".

Reply:
Added that as well, made sense to me.